model/registry/rpc.yaml

```diff
@@ -262,3 +262,19 @@ groups:
         type: int
         stability: experimental
         brief: "Uncompressed size of the message in bytes."
+      - id: rpc.request.body.size
+        type: int
+        stability: experimental
+        brief: "The size of the request payload body in bytes."
+        note: >
+          For gRPC, this value is calculated through [protobuf](https://protobuf.dev/reference/java/api-docs/com/google/protobuf/MessageLite.html#getSerializedSize--)
+          (including unary and streaming calls).
+        examples: 3495
+      - id: rpc.response.body.size
+        type: int
+        stability: experimental
+        brief: "The size of the response payload body in bytes."
+        note: >
+          For gRPC, this value is calculated through [protobuf](https://protobuf.dev/reference/java/api-docs/com/google/protobuf/MessageLite.html#getSerializedSize--)
```

Comment:
```suggestion
          The response body size represents the serialized size of the message payload that is received in the response over the wire. In case of streaming calls, it accounts for the first response payload and does not include size of the consequent message stream.
```